src/main/java/io/jenkins/plugins/luxair/ImageTag.java

```diff
@@ -44,24 +48,60 @@ private ImageTag() {
             .getHeaders().getFirst("Www-Authenticate");
         Unirest.shutDown();
 
-        String pattern = "Bearer realm=\"(\\S+)\",service=\"(\\S+)\"";
-        Matcher m = Pattern.compile(pattern).matcher(headerValue);
-        if (m.find()) {
-            rtn[0] = m.group(1);
-            rtn[1] = m.group(2);
-            logger.info("realm:" + rtn[0] + ": service:" + rtn[1] + ":");
-        } else {
-            logger.warning("No AuthService available from " + url);
+        String type = "";
+
+        String typePattern = "^(\\S+)";
+        Matcher typeMatcher = Pattern.compile(typePattern).matcher(headerValue);
+        if (typeMatcher.find()) {
+            type = typeMatcher.group(1);
+        }
+
+        if (type.equals("Basic")) {
+            rtn[0] = "Basic";
+            logger.info("type:Basic:");
```

Comment:
I know I am not the maintainer of this, but if I may add my two cents here:
Not a big fan of the log format, neither of the original below.

If I may have a suggestion to make it easier decipherable in the logs:
```suggestion
            logger.info("AuthService: type=Basic");
```